src/viur/core/render/vi/__init__.py

```diff
@@ -141,6 +141,8 @@ def canAccess(*args, **kwargs) -> bool:
 
 @exposed
 def index(*args, **kwargs):
+    if args or kwargs:
+        raise errors.NotFound
     if not conf.instance.project_base_path.joinpath("vi", "main.html").exists():
```

Comment:
Alternative solution:
```suggestion
    if args or kwargs or not conf.instance.project_base_path.joinpath("vi", "main.html").exists():
```

Reply:
But then you lose the information about the reason that caused this exception. If the trace exception conf option is activated, you can see the line of the exception and know whether it is an invalid path or whether there is no vi in your deploy folder.